### ISSUE TITLE
Feature Request - Screenshots of secure flag activities

### ISSUE THREAD

**Issue by vithusel** [GrapheneOS/os-issue-tracker#1915]
Giving end users the ability to screenshot apps protected by secure flag would be beneficial. 

Perhaps some additional layer of security would be needed for this (Biometric unlock only or something with a timeout) 

I'm not too sure if a feature like this is likely to get google tightening down on Graphene OS being used with their ecosystem as it does technically circumvent the security of their apps.

**Comment by thestinger** [comment]
> I'm not too sure if a feature like this is likely to get google tightening down on Graphene OS being used with their ecosystem as it does technically circumvent the security of their apps.

It doesn't circumvent anything since you can just take a picture of the screen, use screen recording or an accessibility service.

**Comment by vithusel** [comment]
> > I'm not too sure if a feature like this is likely to get google tightening down on Graphene OS being used with their ecosystem as it does technically circumvent the security of their apps.
> 
> It doesn't circumvent anything since you can just take a picture of the screen, use screen recording or an accessibility service.

Weirdly enough, not too sure if its something new. I tried screen recording but that didn't work either. The screen was black. But i understand what you mean. This isn't something urgent. Just a nice to have. 

Thanks for your amazing work!

**Comment by Juliaria08** [comment]
> > > > I'm not too sure if a feature like this is likely to get google tightening down on Graphene OS being used with their ecosystem as it does technically circumvent the security of their apps.
> > > 
> > > 
> > > It doesn't circumvent anything since you can just take a picture of the screen, use screen recording or an accessibility service.
> > 
> > 
> > Weirdly enough, not too sure if its something new. I tried screen recording but that didn't work either. The screen was black. But i understand what you mean. This isn't something urgent. Just a nice to have.
> > Thanks for your amazing work!
> 
> You can dispaly your Android device on your PC, then take a screenshot of that. This project does that:
> 
> https://github.com/Genymobile/scrcpy

@sergiuser1 I can't seem to be able to take a screenshot of the activity, I have tested it using Whatsapp's one time shown images. And the tool you posted using adb over the network. The behaviour I see is that the screen in scrcpy appears to not show the activity but only show other things (like the statusbar showing the date and battery level)

**Comment by sergiuser1** [comment]
> > > > > I'm not too sure if a feature like this is likely to get google tightening down on Graphene OS being used with their ecosystem as it does technically circumvent the security of their apps.
> > > > 
> > > > 
> > > > It doesn't circumvent anything since you can just take a picture of the screen, use screen recording or an accessibility service.
> > > 
> > > 
> > > Weirdly enough, not too sure if its something new. I tried screen recording but that didn't work either. The screen was black. But i understand what you mean. This isn't something urgent. Just a nice to have.
> > > Thanks for your amazing work!
> > 
> > 
> > You can dispaly your Android device on your PC, then take a screenshot of that. This project does that:
> > https://github.com/Genymobile/scrcpy
> 
> @sergiuser1 I can't seem to be able to take a screenshot of the activity, I have tested it using Whatsapp's one time shown images. And the tool you posted using adb over the network. The behaviour I see is that the screen in scrcpy appears to not show the activity but only show other things (like the statusbar showing the date and battery level)

You're right, if it's screen that blocks screenshots, then it looks black while casting the screen.

**Comment by BreakSecurity** [comment]
@girlbossceo I saw that you have closed this PR. Will this PR be revived or not? Thank you for the information

**Comment by Juliaria08** [comment]
Could the PR that was closed be merged? If so, why hasn't it been merged? if it can't why?
Could I get some status update about this feature?
